### PR TITLE
Add new Landscape Mode feature to App

### PR DESCRIPTION
**Pull request by john-rocky**
Add landscape mode.
The UI will be laid out according to the device orientation and video orientation will be displayed correctly. The detection box will also be displayed correctly. If the app has a preferred landscape UI, please let me know.

## 🛠️ PR Summary

<sub>Made with ❤️ by [Ultralytics Actions](https://github.com/ultralytics/actions)<sub>

### 🌟 Summary
This PR enhances the Ultralytics YOLO iOS app by adding support for human detection and tracking, along with various other improvements and feature additions.

### 📊 Key Changes
- **Model Additions**: Introduced YOLOv8 models for human detection and added relevant `.swift` files (`HumanModel`, `TrackingModel`, and `PostProcessing`).
- **Configuration Changes**: Updated Xcode project configurations and added YOLO app scheme.
- **Orientation Handling**: Improved video orientation handling and updated device orientation settings.
- **UI and Design Updates**: Added landscape support for UI elements, new images, and adjusted some layout constraints.
- **Bounding Box Enhancements**: Improved bounding box handling with `innerTextLayer` for additional information display.
- **Tracking Feature**: Added tracking functionality to persistently identify the same individual across frames.

### 🎯 Purpose & Impact
- **Enhanced Object Detection**: Enables human-specific detection with higher accuracy and relevant information, enriching the app's use-cases.
- **Improved Usability**: Better handling of device orientation, making the app more user-friendly in different device positions.
- **Tracking Capability**: Allows continuous tracking of identified individuals, aiding applications in surveillance and monitoring.
- **UI Improvements**: Provides a more polished and responsive user interface, enhancing the overall user experience.
- **Codebase Organization**: Introduces a more structured codebase making future enhancements and maintenance more manageable.

Overall, this update significantly boosts the app's functionality and usability, catering to both general users and those requiring advanced detection and tracking features. 🚀

### CODE REVIEW

**Comment by john-rocky** [comment]
In landscape mode, the background color of the bottom toolbar has been made transparent so that the preview can be seen more widely.
Also, the share button now works correctly in landscape mode.
Other UI layouts have been kept as original as possible.
If you have any additional layout requests, please let me know and I will make corrections.

IMG_3017

**Comment by pderrenger** [comment]
@john-rocky hello!

Thank you for your detailed feedback and the screenshot. It's great to hear that the background color of the bottom toolbar has been made transparent and that the share button now functions correctly in landscape mode. Your efforts to maintain the original UI layout as much as possible are appreciated.

If you have any specific layout requests or further enhancements you'd like to see, please feel free to share them here. We're always looking to improve the user experience based on valuable input from our community.

If you encounter any issues or bugs, please ensure you provide a minimum reproducible code example. This helps us investigate and resolve the problem more efficiently. You can find more information on how to create one here: [Minimum Reproducible Example](https://docs.ultralytics.com/help/minimum_reproducible_example). Additionally, please verify that you are using the latest versions of `torch` and `ultralytics` to ensure compatibility and access to the latest features and fixes.

Looking forward to your suggestions and feedback!

**Review by ambitious-octopus — COMMENTED** [review]
@john-rocky Landscape mode does not work in iPad 9th generation with iPadOS 17.5.1

**Comment by john-rocky** [comment]
@ambitious-octopus 
I don't have an iPad with me right now, so it would be helpful if you could tell me how it doesn't work.
Does it crash?
Does the layout get messed up?
Does the camera preview have a strange orientation?

etc.

**Review by ambitious-octopus — COMMENTED** [review]
@john-rocky The app simply doesn't rotate when you put the device in landscape mode! Let me test it again and I'll give you more precise information!

**Review by ambitious-octopus — COMMENTED** [review]
I think this is the first PR to be merged, but this PR also contains changes related to YOLO-human. @john-rocky Can you make this PR integrate only landscape mode by removing all changes related to YOLO-human?

**Comment by john-rocky** [comment]
@ambitious-octopus 
OK, I will remove all changes related to YOLO-human.

**Comment by john-rocky** [comment]
@ambitious-octopus 
I removed all changes related to YOLO-human.
Now the landscape mode works only in detect task.
IMG_3424

File: YOLO.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -351,7 +331,7 @@
 				ASSETCATALOG_COMPILER_INCLUDE_ALL_APPICON_ASSETS = NO;
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 0;
-				DEVELOPMENT_TEAM = 3MR4P6CL3X;
+				DEVELOPMENT_TEAM = "";
```
Comment (ambitious-octopus) [inline YOLO.xcodeproj/project.pbxproj:334]: @glenn-jocher, What do you think about this? Should we leave our code or should we make a blank one?

**Review by ambitious-octopus — COMMENTED** [review]
@glenn-jocher This PR is the first step towards integrating into the main branch! While I haven't been able to test it on a Mac due to lack of access, I did test it on an iPhone 15 using TestFlight, and it worked correctly.

File: YOLO.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -13,11 +13,6 @@
 		636EFCAF21E62DD300DE43BC /* VideoCapture.swift in Sources */ = {isa = PBXBuildFile; fileRef = 636EFCA221E62DD300DE43BC /* VideoCapture.swift */; };
 		636EFCB321E62DD300DE43BC /* AppDelegate.swift in Sources */ = {isa = PBXBuildFile; fileRef = 636EFCA721E62DD300DE43BC /* AppDelegate.swift */; };
 		636EFCB921E62E3900DE43BC /* Assets.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 636EFCB821E62E3900DE43BC /* Assets.xcassets */; };
-		6381D2182B7817C200ABA4E8 /* yolov8l.mlpackage in Sources */ = {isa = PBXBuildFile; fileRef = 6381D2132B7817C200ABA4E8 /* yolov8l.mlpackage */; };
```
Comment (ambitious-octopus) [inline YOLO.xcodeproj/project.pbxproj:16]: These lines should not be deleted!

Comment (ambitious-octopus) [reply YOLO.xcodeproj/project.pbxproj:16]: This is because whoever is trying to build the app needs to know where to place the models!

File: YOLO.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -35,11 +30,6 @@
 		636EFCA221E62DD300DE43BC /* VideoCapture.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = VideoCapture.swift; sourceTree = "<group>"; };
 		636EFCA721E62DD300DE43BC /* AppDelegate.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = AppDelegate.swift; sourceTree = "<group>"; };
 		636EFCB821E62E3900DE43BC /* Assets.xcassets */ = {isa = PBXFileReference; lastKnownFileType = folder.assetcatalog; path = Assets.xcassets; sourceTree = "<group>"; };
-		6381D2132B7817C200ABA4E8 /* yolov8l.mlpackage */ = {isa = PBXFileReference; lastKnownFileType = folder.mlpackage; path = yolov8l.mlpackage; sourceTree = "<group>"; };
```
Comment (ambitious-octopus) [inline YOLO.xcodeproj/project.pbxproj:38]: Again, we don't want to eliminate these references!

File: YOLO.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -87,11 +77,6 @@
 		63A946D8271800E20001C3ED /* Models */ = {
 			isa = PBXGroup;
 			children = (
-				6381D2132B7817C200ABA4E8 /* yolov8l.mlpackage */,
```
Comment (ambitious-octopus) [inline YOLO.xcodeproj/project.pbxproj:90]: Again, we don't want to eliminate these references!

File: YOLO.xcodeproj/project.pbxproj [hunk 5]
```diff
@@ -210,13 +195,8 @@
 			isa = PBXSourcesBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
-				6381D21B2B7817C200ABA4E8 /* yolov8m.mlpackage in Sources */,
```
Comment (ambitious-octopus) [inline YOLO.xcodeproj/project.pbxproj:213]: Again, we don't want to eliminate these references!

File: YOLO.xcodeproj/project.pbxproj [hunk 6]
```diff
@@ -210,13 +195,8 @@
...
-				6381D21C2B7817C200ABA4E8 /* yolov8n.mlpackage in Sources */,
 				636EFCAF21E62DD300DE43BC /* VideoCapture.swift in Sources */,
 				636166EA251443B20054FA7E /* ThresholdProvider.swift in Sources */,
-				6381D2182B7817C200ABA4E8 /* yolov8l.mlpackage in Sources */,
```
Comment (ambitious-octopus) [inline YOLO.xcodeproj/project.pbxproj:217]: Again, we don't want to eliminate these references!

File: YOLO.xcodeproj/project.pbxproj (hunk 1 above)
Comment (ambitious-octopus) [inline YOLO.xcodeproj/project.pbxproj:334]: Reading around there is no security issue in keeping DEVELOPMENT_TEAM public, let's leave it like this:
```suggestion
				DEVELOPMENT_TEAM = 3MR4P6CL3X;
```

File: YOLO.xcodeproj/project.pbxproj [hunk 7]
```diff
@@ -379,7 +359,7 @@
 				ASSETCATALOG_COMPILER_INCLUDE_ALL_APPICON_ASSETS = NO;
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 0;
-				DEVELOPMENT_TEAM = 3MR4P6CL3X;
+				DEVELOPMENT_TEAM = "";
```
Comment (ambitious-octopus) [inline YOLO.xcodeproj/project.pbxproj:362]: Same as above it must be:
```suggestion
				DEVELOPMENT_TEAM = 3MR4P6CL3X;
```

File: YOLO.xcodeproj/xcshareddata/xcschemes/YOLO.xcscheme
Comment (ambitious-octopus) [inline YOLO.xcodeproj/xcshareddata/xcschemes/YOLO.xcscheme:1]: Why do we need this file here? Is this file needed to build the app properly?

File: YOLO/Assets.xcassets/Focus.imageset/Contents.json
Comment (ambitious-octopus) [inline YOLO/Assets.xcassets/Focus.imageset/Contents.json:1]: why two new `.png` files were added here when we already have one?

File: YOLO/Assets.xcassets/Focus.imageset/ultralytics_square_focus_image 1.png
Comment (ambitious-octopus) [inline YOLO/Assets.xcassets/Focus.imageset/ultralytics_square_focus_image 1.png:1]: We already have [this](https://github.com/ultralytics/yolo-ios-app/blob/462c36ced989bd707edb7f3a06f367644df3d758/YOLO/Assets.xcassets/Focus.imageset/ultralytics_square_focus_image.png), what is this copy for?

File: YOLO/Assets.xcassets/Focus.imageset/ultralytics_square_focus_image 2.png
Comment (ambitious-octopus) [inline YOLO/Assets.xcassets/Focus.imageset/ultralytics_square_focus_image 2.png:1]: We already have [this](https://github.com/ultralytics/yolo-ios-app/blob/462c36ced989bd707edb7f3a06f367644df3d758/YOLO/Assets.xcassets/Focus.imageset/ultralytics_square_focus_image.png), what is this copy for?

File: YOLO/Info.plist [hunk 8]
```diff
@@ -21,7 +21,7 @@
 	<key>CFBundleShortVersionString</key>
 	<string>$(MARKETING_VERSION)</string>
 	<key>CFBundleVersion</key>
-	<string>24</string>
+	<string>354</string>
```
Comment (ambitious-octopus) [inline YOLO/Info.plist:24]: Why are we changing the build number?

File: YOLO/LaunchScreen.storyboard [hunk 9]
```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="22505" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" launchScreen="YES" useTraitCollections="YES" colorMatched="YES" initialViewController="01J-lp-oVM">
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="32700.99.1234" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" launchScreen="YES" useTraitCollections="YES" colorMatched="YES" initialViewController="01J-lp-oVM">
```
Comment (ambitious-octopus) [inline YOLO/LaunchScreen.storyboard:2]: Why do we need to specify the version here?

File: YOLO/LaunchScreen.storyboard [hunk 10]
```diff
@@ -1,9 +1,9 @@
...
     <device id="retina5_9" orientation="portrait" appearance="light"/>
     <dependencies>
         <deployment identifier="iOS"/>
-        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="22504"/>
+        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="22685"/>
```
Comment (ambitious-octopus) [inline YOLO/LaunchScreen.storyboard:6]: Why do we need to specify the version here?

File: YOLO/Main.storyboard [hunk 11]
```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="22505" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES" initialViewController="BYZ-38-t0r">
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="32700.99.1234" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES" initialViewController="BYZ-38-t0r">
```
Comment (ambitious-octopus) [inline YOLO/Main.storyboard:2]: Why do we need to specify the version here?

File: YOLO/Main.storyboard [hunk 12]
```diff
@@ -1,9 +1,9 @@
...
     <device id="retina6_12" orientation="portrait" appearance="light"/>
     <dependencies>
         <deployment identifier="iOS"/>
-        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="22504"/>
+        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="22685"/>
```
Comment (ambitious-octopus) [inline YOLO/Main.storyboard:6]: Why do we need to specify the version here?

**Review by ambitious-octopus — CHANGES_REQUESTED** [review]
Hey @john-rocky , I added some comments! Have you tested this branch?

File: YOLO.xcodeproj/project.pbxproj (hunk 2 above)
Comment (john-rocky) [reply YOLO.xcodeproj/project.pbxproj]: I changed those lines back and restored the model references.

File: YOLO.xcodeproj/project.pbxproj (hunk 3 above)
Comment (john-rocky) [reply YOLO.xcodeproj/project.pbxproj]: I changed those lines back and restored the model references.

File: YOLO.xcodeproj/project.pbxproj (hunk 4 above)
Comment (john-rocky) [reply YOLO.xcodeproj/project.pbxproj]: I changed those lines back and restored the model references.

File: YOLO.xcodeproj/project.pbxproj (hunk 5 above)
Comment (john-rocky) [reply YOLO.xcodeproj/project.pbxproj]: I changed those lines back and restored the model references.

File: YOLO.xcodeproj/project.pbxproj (hunk 6 above)
Comment (john-rocky) [reply YOLO.xcodeproj/project.pbxproj]: I changed those lines back and restored the model references.

File: YOLO/Info.plist (hunk 8 above)
Comment (john-rocky) [reply YOLO/Info.plist]: I reverted build number.

File: YOLO/LaunchScreen.storyboard (hunk 9 above)
Comment (john-rocky) [reply YOLO/LaunchScreen.storyboard]: The version of the xcode toolbox I use has changed because it is the latest version. I changed it back to the original version just to be safe.

File: YOLO/LaunchScreen.storyboard (hunk 10 above)
Comment (john-rocky) [reply YOLO/LaunchScreen.storyboard]: I changed it back to the original version just to be safe.

File: YOLO/Main.storyboard (hunk 11 above)
Comment (john-rocky) [reply YOLO/Main.storyboard]: I changed it back to the original version just to be safe.

File: YOLO/Main.storyboard (hunk 12 above)
Comment (john-rocky) [reply YOLO/Main.storyboard]: I changed it back to the original version just to be safe.

File: YOLO.xcodeproj/project.pbxproj (hunk 1 above)
Comment (john-rocky) [reply YOLO.xcodeproj/project.pbxproj]: I reverted DEVELOPMENT_TEAM to original.

File: YOLO.xcodeproj/project.pbxproj (hunk 7 above)
Comment (john-rocky) [reply YOLO.xcodeproj/project.pbxproj]: I reverted DEVELOPMENT_TEAM to original.

File: YOLO/Assets.xcassets/Focus.imageset/Contents.json
Comment (john-rocky) [reply YOLO/Assets.xcassets/Focus.imageset/Contents.json:1]: I deleted two new .png files.

File: YOLO/Assets.xcassets/Focus.imageset/ultralytics_square_focus_image 1.png
Comment (john-rocky) [reply YOLO/Assets.xcassets/Focus.imageset/ultralytics_square_focus_image 1.png:1]: I deleted this new .png file.

File: YOLO/Assets.xcassets/Focus.imageset/ultralytics_square_focus_image 2.png
Comment (john-rocky) [reply YOLO/Assets.xcassets/Focus.imageset/ultralytics_square_focus_image 2.png:1]: I deleted this new .png file.

File: YOLO.xcodeproj/xcshareddata/xcschemes/YOLO.xcscheme
Comment (john-rocky) [reply YOLO.xcodeproj/xcshareddata/xcschemes/YOLO.xcscheme:1]: I think we need xcscheme file to build.

What is an xcscheme File?
An xcscheme file is a configuration file in Xcode projects that defines various settings related to building, running, testing, and archiving your application. It determines how different targets within your project should be handled during these processes. Each scheme can be tailored to suit specific needs, such as development, testing, or release builds.

Key Roles of an xcscheme File
Managing Build Settings:

Target Selection: The scheme specifies which targets should be built. Projects often contain multiple targets, such as the main app, test suites, or additional components like widgets.
Build Configuration: It allows you to specify build configurations, such as Debug or Release, which affect optimization levels and debugging information.
Managing Run Settings:

Execution Environment: The scheme defines the environment in which the app will run, including environment variables, command-line arguments, and whether to run on a simulator or physical device.
Debugging Options: It configures whether debugging tools are enabled and which debugger should be used.
Managing Test Settings:

Test Target Selection: The scheme specifies which test targets or test suites should be executed.
Test Environment: It allows configuration of specific environment variables or settings required during testing.
Managing Archive Settings:

Archive Process Configuration: The scheme defines how to archive the app, including which build settings and actions are used. This is crucial for creating release builds or distributing the app via Ad Hoc methods.

**Comment by john-rocky** [comment]
@ambitious-octopus 
I reverted the changes in project.pbxproj.
And I have tested this branch and it works fine.

File: YOLO.xcodeproj/xcshareddata/xcschemes/YOLO.xcscheme
Comment (ambitious-octopus) [reply YOLO.xcodeproj/xcshareddata/xcschemes/YOLO.xcscheme:1]: understood, thanks!

File: YOLO/LaunchScreen.storyboard (hunk 9 above)
Comment (ambitious-octopus) [reply YOLO/LaunchScreen.storyboard]: What are the risks of using an old version here?

File: YOLO/LaunchScreen.storyboard (hunk 10 above)
Comment (ambitious-octopus) [reply YOLO/LaunchScreen.storyboard]: What are the risks of using an old version here?

File: YOLO/Main.storyboard (hunk 11 above)
Comment (ambitious-octopus) [reply YOLO/Main.storyboard]: What are the risks of using an old version here?

File: YOLO/Main.storyboard (hunk 12 above)
Comment (ambitious-octopus) [reply YOLO/Main.storyboard]: Again, What are the risks of using an old version here?

File: YOLO/Info.plist (hunk 8 above)
Comment (ambitious-octopus) [reply YOLO/Info.plist]: Is there any problem if you publish an app with the same build number as the previous one?

File: YOLO/Main.storyboard (hunk 12 above)
Comment (john-rocky) [reply YOLO/Main.storyboard]: Ensuring compatibility:

The toolsVersion indicates which version of Xcode your code was created or updated in. This is important to ensure compatibility when certain features or settings depend on a specific Xcode version.

Supporting new features:

A new toolsVersion allows your storyboard to support new iOS and Xcode features (for example, the latest UI elements or constraint optimizations).

Compatibility issues with older versions:

A new toolsVersion might prevent your storyboard from opening or displaying properly in older versions of Xcode. This happens if the older version doesn't support the new features or settings.

Impact on the build process:

A new toolsVersion might change the build settings and options used in the build process. This might result in build errors or warnings.

Solutions and recommendations
Use the latest version of Xcode:

Make sure everyone on your development team uses the same (latest) Xcode version to avoid issues caused by toolsVersion differences.

Leverage a version control system:

Use a version control system such as Git to see why the toolsVersion was changed in the history and revert the change if necessary.

Communicate with your team:

Changing the toolsVersion can affect the entire project, so communicate the change to your team and make sure everyone is aware of the latest version.

Test and verify:

After changing the toolsVersion, be sure to thoroughly test your app to ensure it works as expected. It is especially important to check that it works on older devices and OS versions.

File: YOLO/Main.storyboard (hunk 11 above)
Comment (john-rocky) [reply YOLO/Main.storyboard]: Ensuring compatibility:

The toolsVersion indicates which version of Xcode your code was created or updated in. This is important to ensure compatibility when certain features or settings depend on a specific Xcode version.

Supporting new features:

A new toolsVersion allows your storyboard to support new iOS and Xcode features (for example, the latest UI elements or constraint optimizations).

Compatibility issues with older versions:

A new toolsVersion might prevent your storyboard from opening or displaying properly in older versions of Xcode. This happens if the older version doesn't support the new features or settings.

Impact on the build process:

A new toolsVersion might change the build settings and options used in the build process. This might result in build errors or warnings.

Solutions and recommendations
Use the latest version of Xcode:

Make sure everyone on your development team uses the same (latest) Xcode version to avoid issues caused by toolsVersion differences.

Leverage a version control system:

Use a version control system such as Git to see why the toolsVersion was changed in the history and revert the change if necessary.

Communicate with your team:

Changing the toolsVersion can affect the entire project, so communicate the change to your team and make sure everyone is aware of the latest version.

Test and verify:

After changing the toolsVersion, be sure to thoroughly test your app to ensure it works as expected. It is especially important to check that it works on older devices and OS versions.

File: YOLO/LaunchScreen.storyboard (hunk 10 above)
Comment (john-rocky) [reply YOLO/LaunchScreen.storyboard]: Ensuring compatibility:

The toolsVersion indicates which version of Xcode your code was created or updated in. This is important to ensure compatibility when certain features or settings depend on a specific Xcode version.

Supporting new features:

A new toolsVersion allows your storyboard to support new iOS and Xcode features (for example, the latest UI elements or constraint optimizations).

Compatibility issues with older versions:

A new toolsVersion might prevent your storyboard from opening or displaying properly in older versions of Xcode. This happens if the older version doesn't support the new features or settings.

Impact on the build process:

A new toolsVersion might change the build settings and options used in the build process. This might result in build errors or warnings.

Solutions and recommendations
Use the latest version of Xcode:

Make sure everyone on your development team uses the same (latest) Xcode version to avoid issues caused by toolsVersion differences.

Leverage a version control system:

Use a version control system such as Git to see why the toolsVersion was changed in the history and revert the change if necessary.

Communicate with your team:

Changing the toolsVersion can affect the entire project, so communicate the change to your team and make sure everyone is aware of the latest version.

Test and verify:

After changing the toolsVersion, be sure to thoroughly test your app to ensure it works as expected. It is especially important to check that it works on older devices and OS versions.

File: YOLO/LaunchScreen.storyboard (hunk 9 above)
Comment (john-rocky) [reply YOLO/LaunchScreen.storyboard]: Ensuring compatibility:

The toolsVersion indicates which version of Xcode your code was created or updated in. This is important to ensure compatibility when certain features or settings depend on a specific Xcode version.

Supporting new features:

A new toolsVersion allows your storyboard to support new iOS and Xcode features (for example, the latest UI elements or constraint optimizations).

Compatibility issues with older versions:

A new toolsVersion might prevent your storyboard from opening or displaying properly in older versions of Xcode. This happens if the older version doesn't support the new features or settings.

Impact on the build process:

A new toolsVersion might change the build settings and options used in the build process. This might result in build errors or warnings.

Solutions and recommendations
Use the latest version of Xcode:

Make sure everyone on your development team uses the same (latest) Xcode version to avoid issues caused by toolsVersion differences.

Leverage a version control system:

Use a version control system such as Git to see why the toolsVersion was changed in the history and revert the change if necessary.

Communicate with your team:

Changing the toolsVersion can affect the entire project, so communicate the change to your team and make sure everyone is aware of the latest version.

Test and verify:

After changing the toolsVersion, be sure to thoroughly test your app to ensure it works as expected. It is especially important to check that it works on older devices and OS versions.

File: YOLO/Info.plist (hunk 8 above)
Comment (john-rocky) [reply YOLO/Info.plist]: If we have published our app on App Store Connect and we want to publish code changes as an update, we need to change the version or build.

Comment (ambitious-octopus) [reply YOLO/Info.plist]: I think in this case adding a +1 to this number is the best solution since once this PR is merged, we publish the update.

File: YOLO/LaunchScreen.storyboard (hunk 9 above)
Comment (ambitious-octopus) [reply YOLO/LaunchScreen.storyboard]: Perfect, let's keep the new version then!

File: YOLO/LaunchScreen.storyboard (hunk 10 above)
Comment (ambitious-octopus) [reply YOLO/LaunchScreen.storyboard]: Again, let's keep the new version then!

File: YOLO/Main.storyboard (hunk 12 above)
Comment (ambitious-octopus) [reply YOLO/Main.storyboard]: Again, keep the new version then!

File: YOLO/Assets.xcassets/Focus.imageset/Contents.json
Comment (ambitious-octopus) [inline YOLO/Assets.xcassets/Focus.imageset/Contents.json:1]: Why was the image moved to 1X scale while it was previously in 3X scale?

File: YOLO/Info.plist
Comment (ambitious-octopus) [inline YOLO/Info.plist:1]: Add +1, so 25!

**Review by ambitious-octopus — COMMENTED** [review]
@john-rocky Please keep the latest versions of plugins. Can you test it please?

File: YOLO/Info.plist
Comment (john-rocky) [reply YOLO/Info.plist:1]: I updated bundle version to 25

File: YOLO/Assets.xcassets/Focus.imageset/Contents.json
Comment (john-rocky) [reply YOLO/Assets.xcassets/Focus.imageset/Contents.json:1]: My mistake. I changed it back to 3x.

**Comment by john-rocky** [comment]
@ambitious-octopus 
I updated the plugin to the latest version, tested it on my iPhone and everything worked fine.

File: YOLO/Info.plist
Comment (ambitious-octopus) [reply YOLO/Info.plist:1]: Nice!

**Review by ambitious-octopus — CHANGES_REQUESTED** [review]

**Comment by glenn-jocher** [comment]
@john-rocky @ambitious-octopus hi guys! I implemented the swift formatting in `main` and in this branch, but for some reason GitHub has problems reconciling the two and says there are conflicts. I've gone ahead and resolved the conflicts in this PR, but I see that it is now showing conflicts in all open PRs (!!). 

What a mess. I meant to simplify things but it looks like I've really created a problem now with the other PRs. 

@ambitious-octopus let's sync up on the YOLO meeting tomorrow to discuss next steps. 

This PR seems good to go now, so I'm going to merge, and then let's focus on the next PRs here. 

@john-rocky I think let's wait on the Human PRs for now, we want to focus first on the simplest improvements to the base app, and then begin to add Tasks like Segment and Pose, but we want to do this an minimally as possible, as users are expecting this PR to be a simple place to start, so we do not want to overdo it with code and features as much as possible.

**Comment by glenn-jocher** [comment]
@john-rocky @ambitious-octopus PR merged!! Thanks for the amazing effort on this!
